helmfile.d/stacks/fluentd.yaml

```diff
@@ -75,6 +75,14 @@ templates:
       - fluentd-system/podsecuritypolicy
       - kube-system/workload-cluster-np
       - monitoring/kube-prometheus-stack # creates servicemonitors and prometheusrules
+    hooks:
```

Comment:
Cool, didn't know you could do this!

Reply:
Yeah, I thought it would be rather strange to use but it turns out it is quite simple to work with.
We could potentially add a CRD update hook in the future to ensure that they always get updated in the future.